erpnext/stock/doctype/pick_list_item/pick_list_item.json

```diff
@@ -21,6 +21,7 @@
   "stock_uom",
   "serial_no_and_batch_section",
   "serial_no",
+  "coa_batch_no",
```

Comment:
- this field should be in the right column.
- Move the package tag field to left column and coa to right one.